src/surfing_penguin/models.py

```diff
@@ -39,52 +39,51 @@ def check_password(self, password):
 class Survey(Base):
     __tablename__ = 'survey'
     id = Column(Integer, primary_key=True)
-    surveyname = Column(String(128))
+    survey_title = Column(String(128))
+    survey_content = Column(String(128))  # Briefly describe it
     question_num = Column(Integer)
+    is_anonymous = Column(Integer, default=0)
     answerlist_num = Column(Integer)
     author_id = Column(Integer, ForeignKey('user.id'))
-    is_anonymous = Column(Integer, default=0)
     questions = relationship("Question")
     answerlists = relationship("AnswerList")
 
-    def __init__(self, user, name, is_anonymous):
-        self.surveyname = name
+    def __init__(self, user, name, content, is_anonymous):
+        self.survey_title = name
         self.author_id = user.id
+        self.survey_content = content
         self.question_num = 0
         self.answerlist_num = 0
         self.is_anonymous = is_anonymous
 
 
 class Question(Base):
-    # TODO: add detail in what should in a question
     __tablename__ = 'question'
     id = Column(Integer, primary_key=True)
+    index_in_survey = Column(Integer)  # NO. in that survey
     title = Column(String(128))
     content = Column(String(1024))
+    format = Column(String(128))
+    choice_num = Column(Integer)
     survey_id = Column(Integer, ForeignKey('survey.id'))
-    index_in_survey = Column(Integer)  # NO. in that survey
-    # TODO: add answer type
 
-    def __init__(self, title, content, survey):
+    def __init__(self, title, content, format, choice_num, survey):
```

Comment:
Could you add some restrictions (maybe with try & exception) when creating a question?